### DISCUSSION TITLE
How to change font type ?

### DISCUSSION THREAD

**Discussion by mide8915** [OpenRCT2/OpenRCT2#17049] [Q&A]
When using simplified Chinese, there is display issuea  with the English characters，it doesn't look very clear. How to fix this by modifying the font files？

**Comment by huamingye** [comment]
我也很想知道……

**Comment by telk5093** [comment]
You can find and change ``[font]`` settings in ``(Document)/OpenRCT2/config.ini`` file manually:
```
[font]
file_name = ""
font_name = ""
x_offset = 0
y_offset = 1
size_tiny = 1
size_small = 0
size_medium = 0
size_big = 0
height_tiny = 0
height_small = 0
height_medium = 0
height_big = 0
enable_hinting = true
hinting_threshold = 0
```

**Reply by huamingye** [reply]
Is this file available in Windows system? I not find ie😵‍💫

**Reply by telk5093** [reply]
Yes, Find ``C:\Users\(Your Windows Account name)\Documents\OpenRCT2\config.ini``

**Reply by huamingye** [reply]
thanks ！let me try 😆

**Reply by huamingye** [reply]
I try to wright like this
[font]
file_name = "msyhbd.ttc"
font_name = ""

but。。。
![微信图片_20241027215032](https://github.com/user-attachments/assets/007962ab-264d-456d-9463-578fedd1737a)

the font “msyhbd.ttc” in the document： C:\Windows\Fonts

**Reply by huamingye** [reply]
not only that， if change others font file name，nothing has changed in the game…… 
![0025E79D](https://github.com/user-attachments/assets/e6d961d9-a1b0-4b12-96f8-6c3a568b755f)

**Reply by huamingye** [reply]
if like this：file_name = "msyhbd.ttc"

It seems that all Asian countries have problems……

**Reply by huamingye** [reply]
I've found that happens with all *.ttc

**Reply by fidwell** [reply]
You also have to change the font size in the settings. This worked for me:

```
[font]
file_name = "msyhbd.ttc"
font_name = ""
x_offset = 0
y_offset = -1
size_tiny = 6
size_small = 10
size_medium = 14
size_big = 18
```

(You might have to try different sizes since these look a little too big to me)

![image](https://github.com/user-attachments/assets/dd18a550-dd48-44c4-a12f-cdfd1563febd)

**Reply by huamingye** [reply]
I got it！Thanks！
But I tried it and it seems that only the *.ttc format font works, and the Chinese display is not very good
